backend/controllers/trainer.controller.js

```diff
@@ -0,0 +1,48 @@
+
+
+
+const testdata = require('../db/testdata.json');
+const trainerController = {

```

Comment:
You should not have these functions inside an object.
Instead, create separate functions.
e.g : 

const getTrainers = () => {....}
const xyz = () => {.....}